base/file.jl

```diff
@@ -356,12 +339,33 @@ is an open file object for this path.
 mktemp(parent)
 
 """
-    mktempdir(parent=tempdir())
+    mktempdir(parent=tempdir(); prefix="$temp_prefix")
```

Comment:
`"$temp_prefix"` == `temp_prefix`

Reply:
The idea is to use the value of the global variable rather than repeating it. Seems to work as expected:
```julia
julia> x = 1
1

julia> """x = "$x" """
       x
x

help?> x

  x = "1" 
```

Reply:
My mistake --- I didn't notice this was in a doc string :)

Reply:
Technically `$(repr(temp_prefix))` is more correct in case there are weird characters in the prefix (seems unlikely, but it is possible).